File: Module_3/castudy2/back-end/src/main/java/model/Employee.java
package model;

public class Employee {
    private int id;
    private String name;
    private String dob;
    private String idCard;
    private double salary;
    private String phoneNumber;
    private String email;

    public Employee(int id, String name, String dob, String idCard, double salary, String phoneNumber, String email) {
        this.id = id;
        this.name = name;
        this.dob = dob;
        this.idCard = idCard;
        this.salary = salary;
        this.phoneNumber = phoneNumber;
        this.email = email;
    }
    public Employee(){

    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDob() {
        return dob;
    }

    public void setDob(String dob) {
        this.dob = dob;
    }

    public String getIdCard() {
        return idCard;
    }

    public void setIdCard(String idCard) {
        this.idCard = idCard;
    }

    public double getSalary() {
        return salary;
    }

    public void setSalary(double salary) {
        this.salary = salary;
    }

    public String getPhoneNumber() {
        return phoneNumber;
    }

    public void setPhoneNumber(String phoneNumber) {
        this.phoneNumber = phoneNumber;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }
}

File: Module_5/caseStudy-comlete/src/app/models/employee.ts
import {EducationDegree} from "./education-Degree";
import {Division} from "./division";
import {Position} from "./position";

export class Employee {
  id: number;
  name: string;
  position: Position;
  educationDegree: EducationDegree;
  division: Division;
  dateOfBirth: string;
  idCard: string;
  salary: number;
  phoneNumber: string;
  email: string;
  address: string;

  constructor(id: number, name: string, position: Position, educationDegree: EducationDegree, division: Division,
              dateOfBirth: string, idCard: string, salary: number, phoneNumber: string, email: string, address: string) {
    this.id = id;
    this.name = name;
    this.position = position;
    this.educationDegree = educationDegree;
    this.division = division;
    this.dateOfBirth = dateOfBirth;
    this.idCard = idCard;
    this.salary = salary
    this.phoneNumber = phoneNumber;
    this.email = email;
    this.address = address;
  }
}
